packages/embark-api/src/server.ts

```diff
@@ -76,6 +87,11 @@ export default class Server {
         return reject(new Error(message));
       }
 
+      // close any open sockets
+      for (const socket of this.openSockets.values()) {
+        socket.destroy();
+      }
```

Comment:
How about

```
this.openSockets.values().forEach(socket => socket.destroy());
``` 
?

Reply:
Updated. I also added a typing to the `Set` elements (`Set<net.Socket>`) for additional clarity.